apps/peer_exchange_tester/peer_exchange_tester.nim

```diff
@@ -0,0 +1,86 @@
+import chronos, chronicles, options, os
+import libp2p/peerId
+import eth/p2p/discoveryv5/enr
+import
+  waku/[
+    waku_node,
+    waku_peer_exchange,
+    node/peer_manager,
+    node/peer_manager/peer_manager,
+    node/peer_manager,
+    factory/waku,
+    factory/external_config,
+    common/logging,
+  ]
+
+proc main() {.async.} =
+  echo "--------------- main started ---------------"
+
+  var wakuConf: WakuNodeConf
+  wakuConf.logLevel = logging.LogLevel.DEBUG
+  wakuConf.logFormat = logging.LogFormat.TEXT
+  wakuConf.staticNodes =
+    @[
+      "/ip4/178.128.141.171/tcp/30303/p2p/16Uiu2HAkykgaECHswi3YKJ5dMLbq2kPVCo89fcyTd38UcQD6ej5W"
+    ]
+  wakuConf.maxConnections = 100
+  wakuConf.pubsubTopics = @["/waku/2/rs/0/0"]
+  wakuConf.clusterId = 1
+  wakuConf.nat = "extip:117.99.49.10"
+  wakuConf.relayPeerExchange = true
+  wakuConf.peerExchange = true
+
+  var wakuApp = Waku.init(wakuConf).valueOr:
+    error "Waku initialization failed", error = error
+    quit(QuitFailure)
+
+  (waitFor startWaku(addr wakuApp)).isOkOr:
+    error "Starting waku failed", error = error
+    quit(QuitFailure)
+
+  let switch = newStandardSwitch()
+  discard switch.start()
+
+  let addrs = "/ip4/178.128.141.171/tcp/30303/"
+  let id = "16Uiu2HAkykgaECHswi3YKJ5dMLbq2kPVCo89fcyTd38UcQD6ej5W"
```

Comment:
Instead of having a hard-coded value, it should support that from a cli parameter. Nevertheless, let's leave it as is for now and optimize in the future if needed.